### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by mbtools**
Initial version of documentation

### CODE REVIEW

**Comment by larshp** [comment]
nice!

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Installation
 
-<!-- TODO: write document
+Exercism let's you program and test your code ABAP directly in your browser. No additional installation is required. However, if you prefer to code and test on your own system, you can get an ABAP trial system.
```
Comment (g-back) [inline docs/INSTALLATION.md:3]:
```suggestion
Exercism let's you program and test your ABAP code directly in your browser. No additional installation is required. However, if you prefer to code and test on your own system, you can get an ABAP trial system.
```

File: docs/TESTS.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+Once you have code you want to test, select "Run Tests". This will trigger the execution of a corresponding ABAP test class and the results will be displayed after a few seconds. If there are errors, go back to your code and adjust it accordingly. If all tests pass, congratulations! You completed the exercise and can submit your solution.
 
-  This document can also link to the testing framework documentation.
+### Troubleshooting
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/tests`.
+Behind the scenes, the your ABAP code is compiled and executed using the so-called [abaplint transpiler](https://github.com/abaplint/transpiler). This is still a relatively new solution. If you have any problems you can't resolve, please create an issue on [GitHub](https://github.com/abaplint/transpiler/issues).
```
Comment (g-back) [inline docs/TESTS.md:11]:
```suggestion
Behind the scenes, your ABAP code is compiled and executed using the so-called [abaplint transpiler](https://github.com/abaplint/transpiler). This is still a relatively new solution. If you have any problems you can't resolve, please create an issue on [GitHub](https://github.com/abaplint/transpiler/issues).
```

File: docs/TESTS.md [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+## On Your Own System
+
+Start your ABAP Development Tools (or transaction SE24 if you have an older system) and create a new ABAP class named like the class in the exercise. Implement the class and implement the test class as well by copying the test code from the Exercism website. You can then run the unit tests locally. 
+
+### Troubleshooting
+
+When codeing and testing in your own system, the standard SAP ABAP compiler and execution is used. If your tests fail, it's likely related to your own code. Take another look.
```
Comment (g-back) [inline docs/TESTS.md:19]:
```suggestion
When coding and testing in your own system, the standard SAP ABAP compiler and execution is used. If your tests fail, it's likely related to your own code. Take another look. 
```

**Review by g-back — COMMENTED** [review]
some small grammar notes

**Comment by mbtools** [comment]
i did this quick. thanks for the corrections 👍

**Comment by larshp** [comment]
@g-back any additional comments?

**Comment by g-back** [comment]
I'll look over it again later today

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+Exercism let's you program and test your ABAP code directly in your browser. No additional installation is required. However, if you prefer to code and test on your own system, you can get an ABAP trial system.
 
-  This document should describe what the student needs to install
-  to allow working on the track on their local system using the CLI.
+Of course, your employer might be running SAP. Please check with your IT department if and how you can get access to an ABAP development sandbox.
 
-  You can include the installation instructions in this document, but
-  usually it is better to link to a resource with the official installation
-  instructions, to prevent the instructions from becoming outdated.
+## Which version to chose?
```
Comment (g-back) [inline docs/INSTALLATION.md:7]:
```suggestion
## Which version to choose?
```

**Comment by g-back** [comment]
Fixed another minor spelling error, looks good otherwise for starters I think. Can always change things later if needed 🙂

**Comment by larshp** [comment]
cool, merging

**Pull request merged**